src/main/java/uk/gov/hmcts/reform/managecase/ApplicationParams.java

```diff
@@ -18,6 +18,12 @@
     private List<String> ccdDataStoreAllowedUrls;
     @Value("${ccd.data-store.allowed-service}")
     private String ccdDataStoreAllowedService;
+    @Value("${notify.email-template-id}")
+    private String emailTemplateId;
+    @Value("${notify.reply-to-email-id}")
+    private String replyToEmailId;
+    @Value("${notify.api-key}")
```

Comment:
empty line before annotation for readability, please.

Reply:
done